### ISSUE TITLE
Add detailed status indicators for ongoing recruitment Apply buttons

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#4934]

**Comment by mnigh** [comment]
@tristan-orourke or @gobyrne is this issue still relevant?

**Comment by tristan-orourke** [comment]
I imagine its still a low-priority nice-to-have

**Comment by mnigh** [comment]
> Gray would like to hold this until Job Poster redesign so that we build a consistent experience across both.

Thanks for the clarification @tristan-orourke. Should this issue still be blocked by "Gray would like to hold this until Job Poster redesign so that we build a consistent experience across both" or can the blocked label and this warning be removed?
